addons/mail/tests/discuss/test_discuss_binary_controller.py

```diff
@@ -0,0 +1,134 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo
+from odoo.addons.mail.tests.test_binary_controller import TestBinaryControllerCommon
+
+
+@odoo.tests.tagged("-at_install", "post_install")
+class TestDiscussBinaryControllerCommon(TestBinaryControllerCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.group = cls.env["discuss.channel"].create({"name": "Group", "channel_type": "group"})
+
+    def _execute_options(self, options, data_user, record):
```

Comment:
I'm not a big fan of this `_execute_options` in general.
It would be more clear if each test had its own setup, not each subtest.

For message, it will post a message every sub test, that's not really what we want. We can just do it once.